bindings/matrix-sdk-ffi/src/encryption.rs

```diff
@@ -411,19 +411,55 @@ impl Encryption {
         self.inner.wait_for_e2ee_initialization_tasks().await;
     }
 
-    /// Get the E2EE identity of a user.
+    /// Get the E2EE identity of a user from the crypto store.
     ///
-    /// Returns Ok(None) if this user does not exist.
+    /// Usually, we only have the E2EE identity of a user locally if the user
+    /// is tracked, meaning that we are both members of the same encrypted room.
     ///
-    /// Returns an error if there was a problem contacting the crypto store, or
-    /// if our client is not logged in.
+    /// To get the E2EE identity of a user even if it is not available locally
+    /// use `request_user_identity()`.
```

Comment:
How does the API work overall: there's both `get_user_identity()` (which may return `None`) and `request_user_identity()` (which is guaranteed to return something for rooms where the two users are present). How is the caller supposed to know which one should be called? Call `get_user_identity()` first, and if it returns nothing, try `request_user_identity()`?

If so:

- can we either document it clearly here?
- or provide a unique method that first fetches from the local crypto store, then does the remote query if not found?

Reply:
Both work for me, but if you want to keep the current version, I'd like a comment precising this on `get_user_identity()` at least please :)